.github/workflows/linux-tests.yml

```
@@ -37,7 +37,9 @@ jobs:
            python-version: ${{ matrix.PYTHON_VERSION }}
       - name: Install dependencies
         shell: bash -l {0}
-        run: conda env update --file requirements/environment.yml
+        run: |
+          conda env update --file requirements/environment.yml
+          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --ignore-installed
```

Per https://github.com/jupyter/qtconsole/pull/493#issuecomment-893112222:
```suggestion
          pip install https://github.com/jupyter/jupyter_client/archive/master.zip --force-reinstall
```
and ditto for the others.